plugins/org.jboss.tools.openshift.ui/plugin.xml

```diff
@@ -978,6 +978,34 @@
            commandId="org.eclipse.ui.edit.delete"
            style="push" />
   </menuContribution>
+  <menuContribution
+        allPopups="false"
+        locationURI="popup:org.jboss.tools.openshift.ui.properties.tab.StorageTab">
+     <command
+           commandId="org.jboss.tools.openshift.ui.command.resource.refresh"
+           icon="icons/refresh.gif"
+           id="org.jboss.tools.openshift.ui.command.resource.refresh"
+           style="push"
+           tooltip="Refresh the resource">
```

Comment:
@fbricon Since we have to add one of these per tab, is there any reason for it not to say 'Refresh the storage'; otherwise I would suggest changing them all to just 'Refresh'